apps/condo/domains/ticket/components/BaseTicketForm/index.tsx

```diff
@@ -152,7 +152,7 @@ export const BaseTicketForm: React.FC<ITicketFormProps> = (props) => {
                                                                     label={PhoneLabel}
                                                                     validateFirst
                                                                 >
-                                                                    <PhoneInput/>
+                                                                    <PhoneInput style={{ width: '100%', height: '40px' }}/>
```

Comment:
This changes are don't needed, because after merging of my pull-request #172 this field will be replaced to `ContactEditor` component.

Reply:
Anyway, make a merge after 2 approvals)
👍 Approved